packages/playwright-tools/.npmignore

```diff
@@ -0,0 +1,5 @@
+**/*
+!README.md
+!LICENSE
+!index.d.ts
```

Comment:
There is no `index` in this package.

Reply:
I don't think we'll publish ever, but I'll fix